Packages/Core/Editor/Generator/Templates/Template.cs

```diff
@@ -0,0 +1,160 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace UnityAtoms.Editor
+{
+    public class Template
+    {
+        private struct Data
+        {
+            public string namespaces;
+            public string withNamespace;
+            public string typeName;
+            public string attributes;
+            public string nestedMenu;
+        }
+
+        public const string autoGeneratedWarning =
+@"//------------------------------------------------------------------------------
```

Comment:
We can work on this comment field to make it clearer that it's by Unity Atoms, generated in this version, etc etc, but it's not hugely important right now. Maybe drop a todo here?

Reply:
Good idea, I’ll drop the todo and see if I can do this dynamically because I wouldn’t want to have to change it with every new version.

Reply:
Putting this here so I don't forget it: [PackageInfo.FindForAssetPath](https://docs.unity3d.com/2020.1/Documentation/ScriptReference/PackageManager.PackageInfo.FindForAssetPath.html)(AssetDatabase.GetAssetPath(this));
This _may_ work and give us all kinds of package information to include.